packages/slice-machine/components/AppLayout/Navigation/Menu/Desktop.tsx

```diff
@@ -34,6 +87,7 @@ const Desktop = () => {
         <Logo />
         <ItemsList mt={4} links={navCtx?.links as []} />
         <Box sx={{ position: "absolute", bottom: "3" }}>
+          <UpdateInfo />
```

Comment:
As the NotLoggedIn Component, it's better to let the context choose if the <UpdateInfo /> should appear or not instead of giving this responsability to the component itself.